control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1286,6 +1287,16 @@ func (r *reconciler) reconcileOLM(ctx context.Context, hcp *hyperv1.HostedContro
 
 	p := olm.NewOperatorLifecycleManagerParams(hcp)
 
+	// Check if the defaultSources are disabled
+	operatorHub := &configv1.OperatorHub{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster",
+		},
+	}
+	if err := r.client.Get(ctx, client.ObjectKeyFromObject(operatorHub), operatorHub); err != nil {
```

Comment:
what if error is not found? should we append the error too?

Reply:
If the error is not found means that the operatorHub is being deployed. I think I can append the error to the slice.